### DISCUSSION TITLE
阿里云盘似乎挂了？？

### DISCUSSION THREAD

**Discussion by jumoon2023** [AlistGo/alist#5510] [General]
现象
前台报：failed get objs: failed to list objs: failed to refresh token: Too Many Requests

尝试通过 https://alist.nn.ci/tool/aliyundrive/request  ，log in 重新获取refresh token 网页也报错。

{"code":"InvalidHeader","message":"Header TargetHost is not valid. Must be {domainid}.api.alicloudccp.com"}

@xhofe

**Comment by lovezongzi** [comment]
并没有,我还正常可以访问

**Reply by jumoon2023** [reply]
我的也恢复了
